src/components/ActionBar/ActionBarIcon.tsx

```diff
@@ -19,6 +19,7 @@ export const ActionBarIcon = () => {
     className: cn(iconClassnames),
     'data-testid': 'action-bar-icon',
     onClick,
+    altText: 'chevron',
```

Comment:
I suggest using an aria-hidden on this icon since the word 'chevron' doesn't actually mean a whole lot to people anyway (most people probably don't now what it is)
If you still want an altText, I would suggest 'arrow-down' or something that would mean more to people.